src/main/utils/CourtDetailsUtils.ts

```diff
@@ -22,3 +24,16 @@ export const formatAreasOfLaw = (areasOfLaw: any[]): string => {
 export const filterByDescription = (contacts: any[], filter: string[]): any[] => {
   return contacts.filter((contact) => filter.includes(contact.description.toLowerCase()));
 };
+
+export const filterAdditionalLinks = (additionalLinks: AdditionalLink[], sidebarLocation: string): AdditionalLink[] => {
+  const results: AdditionalLink[] = [];
+  for (const link of additionalLinks) {
+    if (link.location === sidebarLocation) {
+      if (link.description === 'Financial Remedy') {
+        link.isFinancialRemedy = true;
```

Comment:
@KianKwa - I think (as mentioned on Slack) we need to revise this to be more modular. If we start hardcoding in this prefix text, eventually it will be requested on another sidebar request, then another - and will eventually become the norm. I have reached out to the BAs to ask how to handle this (if we could just make the whole financial remedy section a link, or we could instead a prefix column in the data).

Reply:
This now comes from the database 'description' field. No more hard-coded prefix on front-end